apps/condo/domains/notification/schema/SendMessageService.js

```diff
@@ -78,7 +81,27 @@ async function checkSendMessageMeta (type, meta, context) {
         if (required && !value) {
             throw new GQLError({ ...ERRORS.MISSING_VALUE_FOR_REQUIRED_META_ATTRIBUTE, messageInterpolation: { attr } }, context)
         }
+
+        if (attr === 'data' && typeof value === 'object' && schema.data) {
+            const dataSchema = schema.data
+            for (const dataAttr of Object.keys(dataSchema)) {
+                if (dataAttr === 'required' || dataAttr === 'defaultValue') {
+                    continue
+                }
+                const { required: dataRequired } = dataSchema[dataAttr]
+                if (dataRequired && !value[dataAttr]) {
```

Comment:
We should check only `undefined` and `null`?
I think that `0`, `false`, `<empty line>`(?) are valid values